lib/llm/providers/openai.rb

```diff
@@ -43,5 +43,11 @@ def auth(req)
     def response_parser
       LLM::OpenAI::ResponseParser
     end
+
+    def transform_prompt(prompt)
```

Comment:
We should add references to Provider docs, Yard will follow them when generating docs. Eg:
```ruby
##
# @param prompt (see LLM::Provider#transform_prompt)
# @return (see LLM::Provider#transform_prompt)
```